### PR TITLE
Similar Results Language Keys | LPS-104990

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LPS-104990, I wordmsithed most of these, but still have a couple questions I need addressed (asked in the similar resulats slack channel). I figured I'd push it over to you or I might forget about it after the holiday ;)

### CODE REVIEW

**Comment by thektan** [comment]
Thanks Russ!

I don't know if anyone has a better understanding of those 2 fields since they were taken from what was available from elasticsearch. So what you have will probably be ok?

**Comment by rbohl** [comment]
Don't merge it yet. One or two still need some work. I'll clean it up a bit
next week. Feel free to close the pr and reassign the ticket. Thanks, and
Happy Thanksgiving!

**Comment by thektan** [comment]
I'll reassign the ticket back as a reminder you're still adding more to this. Happy Thanksgiving to you as well! 🦃

File: modules/dxp/apps/portal-search-similar-results/portal-search-similar-results-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1,42 +1,42 @@
 advanced-configuration=Advanced Configuration
 analyzer=Analyzer
-analyzer-help=The analyzer that is used to analyze the free form text. Defaults to the analyzer associated with the first field in fields.
+analyzer-help=Specify the analyzer to use on the input document's fields. If left blank, this defaults to the analyzer associated with the first entry in the fields configuration.
 document-entry=Document Entry
-document-entry-help=Represents the document entry that is displayed through this template.
+document-entry-help=Set the document entry that is displayed through this template.
 federated-search-key=Federated Search Key
 federated-search-key-help=Enter the key of an alternate Search this widget is participating on.
```
Comment (oliv-yu) [inline modules/dxp/apps/portal-search-similar-results/portal-search-similar-results-web/src/main/resources/content/Language.properties]: Just to make sure, 'Search' on this line is intentionally capitalized, right?

Comment (rbohl) [reply modules/dxp/apps/portal-search-similar-results/portal-search-similar-results-web/src/main/resources/content/Language.properties]: No, it should probably be lower case.

**Comment by thektan** [comment]
Thanks for these changes @rbohl!

@oliv-yu also made lowercase change https://github.com/thektan/liferay-portal/pull/102/commits/f0b63b2d4bb6bfba363d2eea7354515814d9687f#diff-59b4c3c2464aefd40d2c35ecb355ce94R7 

Running tests at https://github.com/thektan/liferay-portal/pull/102

**Pull request closed without merging**